sdk/test/metrics/histogram_test.cc

```diff
@@ -134,6 +134,66 @@ TEST(Histogram, DoubleCustomBuckets)
   ASSERT_EQ(std::vector<double>({10, 20, 30, 40}), actual.boundaries_);
   ASSERT_EQ(std::vector<uint64_t>({2, 2, 2, 2, 2}), actual.counts_);
 }
+
+TEST(Histogram, DoubleEmptyBuckets)
+{
+  MeterProvider mp;
+  auto m                      = mp.GetMeter("meter1", "version1", "schema1");
+  std::string instrument_unit = "ms";
+  std::string instrument_name = "historgram1";
+  std::string instrument_desc = "histogram metrics";
+
+  std::unique_ptr<MockMetricExporter> exporter(new MockMetricExporter());
+  std::shared_ptr<MetricReader> reader{new MockMetricReader(std::move(exporter))};
+  mp.AddMetricReader(reader);
+
+  std::shared_ptr<HistogramAggregationConfig> config(new HistogramAggregationConfig());
+  config->boundaries_ = {};
+  std::unique_ptr<View> view{
+      new View("view1", "view1_description", instrument_unit, AggregationType::kHistogram, config)};
+  std::unique_ptr<InstrumentSelector> instrument_selector{
+      new InstrumentSelector(InstrumentType::kHistogram, instrument_name, instrument_unit)};
+  std::unique_ptr<MeterSelector> meter_selector{new MeterSelector("meter1", "version1", "schema1")};
+  mp.AddView(std::move(instrument_selector), std::move(meter_selector), std::move(view));
+
+  auto h = m->CreateDoubleHistogram(instrument_name, instrument_desc, instrument_unit);
+
+  h->Record(5, {});
```

Comment:
nit: add some negative double value to be recorded in the test?

Reply:
Histogram only records positive values. Have modified the test to validate that the negative double values get ignored.